integrations/pytorch/torchvision_sparsification.py

```diff
@@ -354,6 +354,18 @@ def train_model(
     return model, val_acc_history
 
 
+def _save_recipe(
+    recipe_manager: ScheduledModifierManager,
+    save_dir: str,
+):
+    recipe_save_name: str = (
+        f"recipe-{time.strftime('%Y-%m-%d_%H:%M:%S', time.localtime())}.yml"
+    )
```

Comment:
currently, save_dirs are either written from scratch or overwritten on every run, so having `recipe.yaml` as the name instead of including a time stamp should be fine here